crates/matrix-store-key/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "matrix-store-key"
```

Staying in line with the other prefixes I'd suggest we do:
```suggestion
name = "matrix-sdk-store-key"
```

Otherwise, if you don't want this to be matrix-sdk bound, I'd ask why have the `matrix-`-prefix either. So alternatively, as it is truly independent from the other crates in this repo, we could make it more generic:

```suggestion
name = "enc-store-key"
```